nox/virtualenv.py

```diff
@@ -39,6 +39,23 @@
 _SYSTEM = platform.system()
 
 
+def find_uv() -> tuple[bool, str]:
+    uv = shutil.which("uv")
+    if uv is not None:
+        return True, uv
+
+    # Look for uv in Nox's environment as well, to handle `pipx install nox[uv]`.
+    with contextlib.suppress(ImportError, FileNotFoundError):
+        from uv import find_uv_bin
+
+        return True, find_uv_bin()
```

Comment:
```suggestion
    # Look for uv in Nox's environment first, such as for `pipx install nox[uv]`.
    with contextlib.suppress(ImportError, FileNotFoundError):
        from uv import find_uv_bin

        return True, find_uv_bin()

    # Otherwise, see if there's a copy on the path
    uv = shutil.which("uv")
    if uv is not None:
        return True, uv
```

This is the part that needs reversing?

Reply:
Yes. Holding back on this one while we're discussing the "stale uv" issue.

Reply:
You have convinced me :) Let's hope this does get fixed in pipx.